recipe/recipe.yaml

```diff
@@ -0,0 +1,254 @@
+schema_version: 1
+
+context:
+  name: ESMValTool
+  version: 2.13.0
+  python_min: 3.11
+  python_max: 3.14
```

Comment:
whoops I missed this one: is max the lowest highest we don't support or is it the highest we support?